fala/apps/adviser/utils.py

```diff
@@ -0,0 +1,46 @@
+# utils.py
+
+CATEGORY_TRANSLATIONS = {
+    "aap": "claims-against-public-authorities",
+    "med": "clinical-negligence",
+    "com": "community-care",
+    "crm": "crime",
+    "deb": "debt",
+    "disc": "discrimination",
+    "edu": "education",
+    "mat": "family",
+    "fmed": "family-mediation",
+    "hou": "housing",
+    "hlpas": "hlpas",
+    "immas": "immigration-asylum",
+    "mhe": "mental-health",
+    "mosl": "modern-slavery",
+    "pl": "prison-law",
+    "pub": "public-law",
+    "wb": "welfare-benefits",
+}
+
+SLUG_TO_CODE = {v: k for k, v in CATEGORY_TRANSLATIONS.items()}
```

Comment:
This is me, but I dislike single character variables, they aren't descriptive and sometime make it harder to read. Using descriptive variable names in loops enhances readability and makes the code more maintainable.

```
SLUG_TO_CODE = {display_name: category_code for category_code, display_name in CATEGORY_TRANSLATIONS.items()}
```